backend/src/Controller/Subscription/SubscriptionController.php

```diff
@@ -186,8 +186,19 @@ public function getSubscriptionForStoreOwner(): JsonResponse
      *      @OA\JsonContent(
      *          @OA\Property(type="string", property="status_code", description="9302"),
      *          @OA\Property(type="string", property="msg", description="You do not have a subscription Successfully."),
-     *          @OA\Property(type="string", property="Data", description="error"),
-     *      )
+     *          @OA\Property(type="object", property="Data",
+     *                @OA\Property(type="integer", property="id"),
+     *                @OA\Property(type="integer", property="packageID"),
+     *                @OA\Property(type="string", property="packageName"),
+     *                @OA\Property(type="integer", property="remainingOrders"),
+     *                @OA\Property(type="integer", property="remainingCars"),
+     *                @OA\Property(type="object", property="status"),
```

Comment:
I guess some fields type annotations need to be corrected, as _status_ and _endDate_.